### PR TITLE
[16.0][FIX] pos_reset_search: TypeError on _clearSearch method.

### PR DESCRIPTION
**Pull request by suker**
It occurs when click on a product using pos in mobile resolution. 
 _toggleMobileSearchbar event is blocked when search input is not null.

### CODE REVIEW

File: pos_reset_search/__manifest__.py
Comment (legalsylvain) [inline pos_reset_search/__manifest__.py]: FYI, this change is not necessary. (done automatically by bot, when merging).

Comment (suker) [reply pos_reset_search/__manifest__.py]: You're right, fixed.

**Review by danielduqma — APPROVED** [review]
LGTM, functional tests also OK

**Comment by suker** [comment]
@OCA/pos-maintainers anyone to move this?

**Comment by suker** [comment]
@OCA/pos-maintainers can we move this fix?

Thanks in advance

**Comment by legalsylvain** [comment]
@Fkawala Do you have the possibility to review this one, before merging it ? 
thanks !

**Review by Fkawala — APPROVED** [review]

**Comment by legalsylvain** [comment]
Hi @Fkawala Thanks for your review ! 

Just a point, please don't merge directly in main OCA branches.
image


instead, use command ``\ocabot merge OPTION``

Thanks !

**Comment by suker** [comment]
Thank you @legalsylvain and @Fkawala!

**Pull request merged**
